src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php

```diff
@@ -42,12 +49,20 @@ class UnusedTagsPass implements CompilerPassInterface
         'kernel.event_subscriber',
         'kernel.fragment_renderer',
         'kernel.locale_aware',
+        'kernel.reset',
+        'mailer.transport_factory',
         'messenger.bus',
-        'messenger.receiver',
         'messenger.message_handler',
+        'messenger.receiver',
+        'messenger.transport_factory',
         'mime.mime_type_guesser',
         'monolog.logger',
-        'proxy',
```

Comment:
Just asking: is it ok to be removed from the list ?

Reply:
fixed